recipes/BEHST/build.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#$ -cwd
+#$ -S /bin/bash
+#
+set -o nounset -o pipefail -o errexit
+set -o xtrace
+
+PREFIX=.
+
+cd $PREFIX/bin/
+
+mkdir $PREFIX/../results/
+
+mkdir $PREFIX/../temp/
+
+
+sh project.sh $PREFIX/../data/pressto_LUNG_enhancers.bed DEFAULT_EQ DEFAULT_ET
```

Comment:
My package on Bitbucket contains data files in the /data/ folder.

What should I do?

Thanks